ksp_plugin_adapter/window_renderer.cs

```diff
@@ -124,6 +124,25 @@ internal abstract class BaseWindowRenderer : ScalingRenderer, IConfigNode {
     }
     UnityEngine.GUI.skin = skin_;
     if (show_) {
+      // N.B.(al2me6): Empirical observation gave the following conclusions:
```

Comment:
Our style would call for `// NOTE(al2me6)`, I think.

Also, maybe "gave" -> "led to".

Thanks for doing this analysis, btw, I can imagine it being rather time-consuming 😀